torch2jax/__init__.py

```diff
@@ -242,15 +256,39 @@ def fn(*args, **kwargs):
 auto_implements(torch.add, jnp.add)
 auto_implements(torch.exp, jnp.exp)
 auto_implements(torch.nn.functional.gelu, jax.nn.gelu)
-auto_implements(torch.mean, jnp.mean)
 auto_implements(torch.mul, jnp.multiply)
 auto_implements(torch.permute, jnp.transpose, dont_coerce_argnums=(1, 2))  # TODO: do we need argnum 2?
 auto_implements(torch.pow, jnp.power)
+auto_implements(torch.sin, jnp.sin)
+auto_implements(torch.cos, jnp.cos)
 auto_implements(torch.sigmoid, jax.nn.sigmoid)
 auto_implements(torch.sqrt, jnp.sqrt)
-auto_implements(torch.sum, jnp.sum)
 auto_implements(torch.tanh, jnp.tanh)
 auto_implements(torch.transpose, jnp.swapaxes)
+auto_implements(torch.rsqrt, lambda x: jnp.reciprocal(jnp.sqrt(x)))
+
+
+@implements(torch.nn.functional.silu)
+def silu(x, *args, **kwargs):
+  x = _v(x)
+  if "inplace" in kwargs and kwargs["inplace"]:
+    assert not kwargs["inplace"], "jax doesn't support inplace"
+  return jax.nn.silu(x)
```

Comment:
please place this in alphabetical order in the torch.nn.functional section below: https://github.com/samuela/torch2jax/blob/7132ab7d7354a12afed82902581180de143aabf3/torch2jax/__init__.py#L519